src/app/utils/reorderable-list/reorderable-list.component.html

```diff
@@ -0,0 +1,19 @@
+<div
+  cdkDropList
+  [cdkDropListOrientation]="orientation"
+  [cdkDropListLockAxis]="dropLockAxis"
+  (cdkDropListDropped)="drop($event)"
+>
+  <div class="flex-row gap-regular flex-wrap gap-large column">
+    @for (item of items; track item) {
+      <div cdkDrag class="drop-item">
+        <ng-container
+          *ngTemplateOutlet="
+            itemTemplate;
+            context: { $implicit: item, remove: remove.bind(this) }
+          "
+        ></ng-container>
```

Comment:
shall we make the template optional and build in a default template already? Or do/should these look very different for the two admin-components? (if not, we may even skip the option to input a template completely for now and only add that later if needed)